README.md

```diff
@@ -40,7 +40,7 @@ Take a look [here](https://github.com/GeyserMC/Geyser/wiki#Setup) for how to set
 - Structure block UI
 
 ## What can't be fixed
-The following things can't be fixed because of Bedrock limitations. They might be fixable in the future, but not as of now.
+We can't fix the following things because of Bedrock's limitations. They might be fixable in the future, but not as of now.
```

Comment:
I would not write "we" because these things cannot be changed by any translation software

Reply:
+1

Reply:
Yes.. that's covered by the "because of Bedrock's limitations"

Reply:
The current wording doesn't place the blame on us exclusively. The listed things cannot be fixed by anyone, nor by anyone except the Bedrock developers.

Reply:
"The following things can only be fixed by Bedrock developers. They might be fixable in the future, but not as of now."

Reply:
Let's not blame the Bedrock developers. 

"The following things cannot be fixed without changes to Bedrock. As of now, they are not fixable in Geyser." or something.

Reply:
```suggestion
The following things cannot be fixed without changes to Bedrock. As of now, they are not fixable in Geyser.
```